### PR TITLE
docs: currect the helm repo address.

### PR DESCRIPTION
**Pull request by sober-wang**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches
- [x] currect documents.

- Related issues
[#649](https://github.com/apache/apisix-ingress-controller/issues/649)

### CODE REVIEW

File: docs/en/latest/deployments/ack.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ As the data plane of apisix-ingress-controller, [Apache APISIX](http://apisix.ap
 
 ```shell
 cd /path/to/apisix-helm-chart
-helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add apisix https://charts.apiseven.com
```
Comment (tao12345666333) [inline docs/en/latest/deployments/ack.md:40]: The original content is correct, we do need to add bitnami repo to install etcd.

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

**Comment by tao12345666333** [comment]
Thanks for your contribution. 

But the content in this PR does not need to be modified, I will close this PR, and you are welcome to participate in trying other content https://github.com/apache/apisix-ingress-controller/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Comment by sober-wang** [comment]
But the incorrect documents will mislead out users.

**Comment by sober-wang** [comment]
But the incorrect documents will mislead our users

**Comment by tao12345666333** [comment]
What’s your meaning?

**Pull request closed without merging**